hack/install-tools.sh

```diff
@@ -52,7 +52,7 @@ if [ -z "$PY_CMD" ]; then
     fi
 fi
 
-ALT_BIN="$(realpath "${2:-.bin}")"
+ALT_BIN="${2:-.bin}"
```

Comment:
I'm not thrilled with this because it worked fine on Linux. It's only on MacOS (and maybe bsd?) that this failed. It appears that realpath on linux doesn't require the path to exist, but Mac does.

At the very least could you mention that this is a fix for *Mac* in the commit message?

Reply:
done, thanks